### DISCUSSION TITLE
Data backup

### DISCUSSION THREAD

**Discussion by jerseyj** [mbridak/FieldDayLogger#30] [Ideas]
I’ve run across similar programs (not cross-platform) that have a function built in to backup the log either by every x qso’s or by time (every 5 minutes say).

This would be very helpful as we all know computers and hard drives can go down at the worst times. 

We configure to backup on the hard drive and on a thumb drive.

KB2GCG

**Comment by mbridak** [comment]
Hey, thanks for the feature request. Sounds like a good idea.

Mike K6GTE

**Comment by jerseyj** [comment]
I mentioned the idea to KM4ACK and he suggested I raise it to you.Jerry KB2GCG - "Defeat lasts one day, giving up lasts a lifetime."On Jun 8, 2023, at 7:09 PM, Michael Bridak ***@***.***> wrote:﻿
Hey, thanks for the feature request. Sounds like a good idea.
Mike K6GTE
